dashboards/resource_lakeview_dashboard.go

```diff
@@ -0,0 +1,179 @@
+package dashboards
+
+import (
+	"context"
+	"crypto/md5"
+	"fmt"
+	"log"
+
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Calculates MD5 hash of the given content
+func calculateMd5Hash(content []byte) string {
+	return fmt.Sprintf("%x", md5.Sum(content))
+}
+
+// Reads content from a JSON string or a file path and returns the content and its MD5 hash
+func readSerializedJsonContent(jsonStr, filePath string) (serJSON string, md5Hash string, err error) {
+	var content []byte
+	if filePath != "" {
+		content, err = common.ReadFileContent(filePath)
+		if err != nil {
+			return "", "", err
+		}
+	} else {
+		log.Printf("[INFO] Reading `serialized_json` of %d bytes", len(jsonStr))
+		content = []byte(jsonStr)
+	}
+	md5Hash = calculateMd5Hash(content)
+	return string(content), md5Hash, nil
+}
```

Comment:
Same for this, it's more common in nature. Maybe right now only lakeview resource will be using this method but it would be nice it we refactor it now to avoid situations like moving `ReadFileContent` to common/util.go that we are doing in this PR in future.

Reply:
+1